feature/exercise-view/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/exerciseview/home/ExerciseScreenTopAppBar.kt

```diff
@@ -235,12 +237,18 @@ internal fun TopBarExerciseInformation(
             val complaintPossible = exercise.bind { exercise ->
```

Comment:
It seems weird that this variable is of type Boolean? even though orElse is used. Can you use exercise.allowComplaintsForAutomaticAssessments ?: false to clarify the code?